src/praisonai-deploy/infra/compose/agents-stack/.env.example

```
@@ -4,9 +4,10 @@
 AGENTS_FILE=./agents.yaml
 API_PORT=8005
 
-# Postgres host binding: keep 127.0.0.1 to avoid exposing the DB on all interfaces.
-POSTGRES_BIND=127.0.0.1
 POSTGRES_PORT=5432
+# Host interface Postgres is published on. Defaults to loopback; only set to
+# 0.0.0.0 when the host network is trusted.
+POSTGRES_BIND=127.0.0.1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_BIND key should go before the POSTGRES_PORT key

(UnorderedKey)